libs/sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -1219,6 +1219,13 @@ public void test_insert_returning() {
         printStatement("insert into foo (id, name) values ('string', 1.2) returning foo.*");
     }
 
+    @Test
+    public void test_using_fqn_in_column_list_of_insert_into_results_in_user_friendly_error() {
+        assertThatThrownBy(() -> printStatement("insert into tbl (t.x) values (1)"))
+            .hasMessage("Column references used in INSERT INTO <tbl> (...) must use the column name. They cannot qualify catalog, schema or table. Got `t.x`");
```

Comment:
not insisting: Maybe also add the `.isExactlyInstanceOf()`

Reply:
The type of exception isn't that important here as no other components rely on it